### ISSUE TITLE
Term Request: Lower eyelid entropion

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11733]
### Term Request: Lower eyelid entropion

**Preferred term label:**
Lower eyelid entropion

**Synonyms:**
* Entropion of the lower eyelid
* Inversion of lower eyelid margin
* Lower lid entropion

**Definition (free text):**
An inward turning (inversion) of the margin of the lower eyelid.

**Parent term (use hpo.jax.org/app):**
Entropion (HP:0000621)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
